Public/Src/Tools/Execution.Analyzer/BxlAnalyzer.dsc

```diff
@@ -43,6 +44,7 @@ namespace Execution.Analyzer {
             importFrom("BuildXL.Ide").Generator.dll,
             importFrom("BuildXL.Utilities").dll,
             importFrom("BuildXL.Utilities").Branding.dll,
+            importFrom("BuildXL.Utilities").Ipc.dll,

```

Comment:
>mportFrom("BuildXL.Utilities").Ipc.dll, [](start = 13, length = 39)

why do you need this?

Reply:
Good question, I think this may have been needed for some field before we nuked because it does not seem to be needed anymore.

---
In reply to: [314835632](https://github.com/microsoft/BuildXL/pull/724#discussion_r314835632) [](ancestors = 314835632)